### PR TITLE
New navbar

### LINKED ISSUE #346: To give an option of logout Button in navbar after successful login of a user.
## I explored this website and found a issue in navbar.

I login successfully with my G-mail account in website but then too navbar shows the option to sign in and sign up.
If i would get an option to log-out there that would be more clear to the user to logout successfully from this application.

### PR DESCRIPTION
**Pull request by ritik-vashist**
fixes #346 

Updated user avatar in  navbar.

### CODE REVIEW

**Comment by ritik-vashist** [comment]
@Abhishek-kumar09  Sir, this  is the new PR for the issue #346 and updated changes of #352.
I request you please check it once.

Screenshot after modifications are attached bellow.

**Comment by ritik-vashist** [comment]
@vasudevsinghal , @Abhishek-kumar09 Sir, I request you to review my work and share your valuable feedback.

File: src/components/NavBar/index.js [hunk 1]
```diff
@@ -103,18 +104,31 @@ function NavBar() {
...
+          {((user === undefined) && (user === null)) ? (
+            <Avatar
+              className={classes.avatar}
+              src="https://media-exp1.licdn.com/dms/image/C560BAQHSXxzxdL45FA/company-logo_200_200/0/1588676153700?e=2159024400&v=beta&t=PEiMBARAmeCUpRrAuJZWev-F_oCZac_OunmUXcLuY5U"
+            />) : (
+            <Avatar
+              className={classes.avatar}
+              src={`${user.photoURL}`}
+            />
+          )}
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:131]: Remove this and put 
```js
<Avatar             
  className={classes.avatar}             
  src={`${user.photoURL}`}   
/>
```
under the condition above.
```jsx
<Typography  className={classes.title} variant="h6" noWrap>
           Hi, {user.displayName}            
</Typography>
```

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Since there are no changes in `package.json` there must be no change in `package-lock.json`.

**Comment by ritik-vashist** [comment]
But @Abhishek-kumar09  Sir, it will be needed in mobile view to show code for cause logo when the user is not log-in.

**Pull request closed without merging**